### PR TITLE
Fix Terraform version check with CLI env vars

### PR DESCRIPTION
**Pull request by awiddersheim**
Fix passing command line arguments to Terraform using the `TF_CLI_ARGS`
environment variables[1]. Need to exclude those variables from the
version check.

[1]: https://www.terraform.io/docs/commands/environment-variables.html#tf_cli_args-and-tf_cli_args_name

### CODE REVIEW

File: cli/version_check.go [hunk 1]
```diff
@@ -25,6 +26,13 @@ func PopulateTerraformVersion(terragruntOptions *options.TerragruntOptions) erro
 	terragruntOptionsCopy.Writer = ioutil.Discard
 	terragruntOptionsCopy.ErrWriter = ioutil.Discard
 
+	// Remove any TF_CLI_ARGS before version checking
```
Comment (brikis98) [inline cli/version_check.go:29]: NIT: could you expand this comment to explain why we're removing these (i.e., these are args to be appended to the "real" command and not the `--version` check) and add a link to the `TF_CLI_ARGS` docs?

**Review by brikis98 — COMMENTED** [review]
Thanks for the PR! This looks like a good fix. Just one suggestion and then I can kick off tests.

File: cli/version_check.go (hunk 1 above)
Comment (awiddersheim) [reply cli/version_check.go]: Sure! Updated.

**Comment by brikis98** [comment]
Great, thank you! One more thing, could you pull in the latest from `master`? I can kick off tests after that.

**Comment by awiddersheim** [comment]
@brikis98 Sure! Rebased.

**Comment by brikis98** [comment]
Thanks! Kicking off tests now.

**Comment by awiddersheim** [comment]
@brikis98 Looks like I didn't `fmt` my last commits. I updated. Can you kick them off again?

**Comment by brikis98** [comment]
Yup!

**Review by brikis98 — APPROVED** [review]
Tests passed! Merging now.

**Comment by brikis98** [comment]
https://github.com/gruntwork-io/terragrunt/releases/tag/v0.26.3

**Comment by awiddersheim** [comment]
Thanks!

**Pull request merged**